src/sonic-host-services/scripts/hostcfgd

```diff
@@ -120,9 +142,9 @@ class FeatureHandler(object):
         self._config_db = config_db
         self._device_config = device_config
         self._cached_config = {}
-        self.is_multi_npu = device_info.is_multi_npu()
+        self.is_multi_npu_device = is_multi_npu()
```

Comment:
What is the motivation for the attribute name change? IMO is_multi_npu is self-desriptive enough, the additional _device does not contribute to readability.

Reply:
Idea was to mock them.

```
hostcfgd.is_multi_npu = mock.Mock(return_value=False)
hostcfgd.get_num_npus = mock.Mock(return_value=1)
```